dev/com.ibm.ws.wsbytebuffer/bnd.bnd

```diff
@@ -0,0 +1,68 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: WsByteBuffer
+Bundle-SymbolicName: com.ibm.ws.wsbytebuffer; singleton:=true
+Bundle-Description: Provides WsByteBuffer buffer functionality
+
+WS-TraceGroup: ChannelFramework
+
+IBM-Default-Config: OSGI-INF/wlp/defaultInstances.xml
+
+# For each exported package, create (in that package) a package-info.java
+# file, and place an @version javadoc tag in package-level javadoc. 
+# Append ";provide:=true" if this bundle also provides an implementation
+# for the exported API.
+#	com.ibm.wsspi.bytebuffer;version="1.0";-split-package:=merge-last
+
+Export-Package: com.ibm.wsspi.bytebuffer;version="1.0";provide:=true, \
+    com.ibm.ws.bytebuffer.internal;version="1.0";provide:=true, \
+    com.ibm.ws.bytebuffer.internal.resources;version="1.0";provide:=true, \
+    com.ibm.websphere.wsbytebuffer.osgi;provide:=true
+
+Include-Resource: \
+    OSGI-INF=resources/OSGI-INF
+
+-dsannotations: \
+    com.ibm.ws.bytebuffer.internal.ByteBufferConfiguration
+
+Service-Component: \
```

Comment:
unused?

Reply:
Testing if removing this affects:
```
    @Reference(service = ByteBufferConfiguration.class,
               cardinality = ReferenceCardinality.MANDATORY)
    protected void setByteBufferConfig(ByteBufferConfiguration bbConfig) {
        wsbbmgr = bbConfig;
    }
```
in CHFBundle
The code in getBufferManager will call -> ChannelFrameworkFactory.getBufferManager -> WsByteBufferPoolManagerImpl.getRef as a fallback if there is no service. Which means the code
above would normally get null but it would work out. 
But I don't think we should remove the @Reference code in case some user feature or OEM
makes use of it by providing such a service??? (why was it there in the first place??).